src/tool/cppwinrt/strings/base_error.h

```diff
@@ -380,10 +380,6 @@ namespace winrt
         {
             return hresult_error(impl::error_fail, to_hstring(e.what())).to_abi();
         }
-        catch (...)
```

Comment:
This ensures that stray exceptions are caught (by noexcept) without unwinding, making for easier debugging.